### PR TITLE
EZP-31875: Implemented JWT authentication Controller

### PR DESCRIPTION
**Pull request by DominikaK**
| Question      | Answer
| ------------- | ---
| JIRA Ticket   | https://jira.ez.no/browse/EZP-31875
| Versions      | master (3.2)

Companion PR for REST reference: https://github.com/ezsystems/ezplatform-rest-reference/pull/10

### CODE REVIEW

File: docs/api/general_rest_usage.md [hunk 1]
```diff
@@ -408,6 +409,27 @@ Accept: application/vnd.ez.api.Root+json
...
+you can get the JWT token through the following request:
+
+```
+POST /user/token/jwt HTTP/1.1
+Host: <yourdomain>
+Accept: application/vnd.ez.api.JWT+xml
+Content-Type: application/vnd.ez.api.JWTInput+xml
+```
+
+Provide the user and password in the request body:
```
Comment (dabrt) [inline docs/api/general_rest_usage.md:424]:
```suggestion
Provide the user name and password in the request body:
```

File: docs/guide/security.md [hunk 2]
```diff
@@ -89,6 +89,53 @@ Symfony provides native support for [multiple user providers](https://symfony.co
 
 See [Authenticating a user with multiple user provider](user_management/user_management.md#authenticating-user-with-multiple-user-providers) for more information.
 
+## JWT authentication
+
+To use [JWT authentication](https://jwt.io/) with eZ Platform, in the provided ` config/packages/lexik_jwt_authentication.yaml` file,
+modify the existing configuration by setting `authorization_header` to `enabled`.:
```
Comment (dabrt) [inline docs/guide/security.md:95]:
```suggestion
modify the existing configuration by setting `authorization_header` to `enabled`:
```

**Review by dabrt — APPROVED** [review]

File: docs/guide/security.md [hunk 3]
```diff
@@ -89,6 +89,53 @@ Symfony provides native support for [multiple user providers](https://symfony.co
...
+    token_extractors:
+        authorization_header:
+            enabled: true
+        cookie:
+            enabled: false
+        query_parameter:
+            enabled: false
+```
+
+You also need new Symfony firewall configuration for REST and GraphQL APIs.
```
Comment (kmadejski) [inline docs/guide/security.md:112]:
```suggestion
You also need a new Symfony firewall configuration for REST and/or GraphQL APIs.
```

**Review by kmadejski — APPROVED** [review]
Just a small suggestion, besides that, looks good to me 👍

**Review by bdunogier — APPROVED** [review]

**Pull request merged**